crates/mockiato-codegen/src/parse/mockable_attr.rs

```diff
@@ -77,3 +50,54 @@ impl MockableAttr {
         })
     }
 }
+
+fn get_meta_items(args: AttributeArgs) -> Result<impl Iterator<Item = Meta>> {
+    let meta_items = args.into_iter().map(|nested| match nested {
+        NestedMeta::Meta(meta) => Ok(meta),
+        NestedMeta::Literal(literal) => Err(unsupported_syntax_error(&literal)),
+    });
+    merge_results(meta_items)
+}
+
+fn attribute_property_not_supported_error(meta_item: &Meta) -> Error {
+    let error_message = format!(
+        "This attribute property is not supported by #[{}]",
+        ATTR_NAME
+    );
+    DiagnosticBuilder::error(meta_item.span(), error_message)
+        .build()
+        .into()
+}
+
+fn static_references_specified_more_than_once_error(meta_item: &Meta) -> Error {
+    let error_message = format!(
+        "`{}` is specified more than once.",
+        STATIC_REFERENCES_ATTR_PARAM_NAME
+    );
+    DiagnosticBuilder::error(meta_item.span(), error_message)
+        .build()
+        .into()
+}
+
+fn name_specified_more_than_once_error(meta_item: &Meta) -> Error {
+    let error_message = format!(
+        "`{}` should only be specified once",
```

Comment:
Why is this error message only slightly different from the one in `static_references_specified_more_than_once_error`? Can't they share the phrasing?

Reply:
Unified in 506a773